### ISSUE TITLE
Just to let you know, popcorn-time.ga seems to be down

### ISSUE THREAD

**Issue by dlevi309** [popcorn-time-ru/popcorn-ru#32]
Thanks for all that you've done and keeping the API server alive

**Comment by ivan1986** [comment]
domain is ok, only 1.1.1.1 issue, maybe some dns update issue
update to latest version from github, it has other domains

**Comment by dlevi309** [comment]
I actually use it for my own personal client! It’s only me (one user), so I thought it would be alright. Either way, the domain is resolving now 👍 I’ll close this out, but thank you for a quick response, here’s a pic of the client too, it’s a fork of someone’s defunct project
![3126B6AB-3ADD-4243-A9AD-CEA5487B3477](https://user-images.githubusercontent.com/7036054/158396720-08520065-c08c-4239-b60f-65f64e4f863c.jpeg)

**Comment by ivan1986** [comment]
the client, as I understand it, is for iOS - there is absolutely nothing clear with them, a bunch of forks are relatively alive to varying degrees, in general, the patient is rather dead

**Comment by dlevi309** [comment]
Yeah, unfortunately, I agree. I used to try to maintain my own personal fork of the main client, but it was just so messy that I doubt that codebase will ever be maintained again. This the project I built my current one off of, it’s not a variation of the old one, it’s one that a developer built from scratch (and it’s very good) I recommend you check it out, it was easy for me to strip some of the old remnants and reconfigure the server info. https://github.com/getpopcorn/graveyard-popcorn
